packages/functions/src/__tests__/xor.test.ts

```diff
@@ -56,13 +56,13 @@ describe('Core Functions / Xor', () => {
       ),
     ).toEqual([
       {
-        message: '"version" and "title" must not be both defined or both undefined',
```

Comment:
it seems like we dropped support for all undefined, is there a reason for this?

Reply:
I'm not sure I understand. The first test on lines 11-27 'given no properties, should return an error message' and the second test on lines 29-45 'given multiple properties that do not match, should return an error message' both still ensure that if all properties are undefined, then an error is returned. (Just with different error text supporting 1-n properties defined instead of just 2.)

Did you mean something else?

(I've committed both of your other change suggestions, thanks!)